app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -290,15 +295,15 @@ const styles = StyleSheet.create({
     marginBottom: 0
   },
 
-  iconLink: {
+  balance__iconLink: {
     textDecoration: 'none',
 
     ':hover': {
       textDecoration: 'none'
     }
   },
 
-  iconText: {
+  balance__iconLink__iconText: {
     color: globalStyles.color.mediumGray,
     margin: '0 0 0 5px',
 
```

Comment:
Actually `paymentStyles.font.regular` should be replaced with `globalStyles.fontSize.settingItemSubtext` as there is no reason why font-size on the payments tab is different than the other tabs.